etc/db-config.yaml

```diff
@@ -114,13 +114,6 @@
             description: Maximum size of error/system output stored in the database (in bytes); use `-1` to disable any limits.
             regex: /^[1-9]\d*$/
```

Comment:
You probably also want to change the regex for `output_storage_limit` then, and maybe refer to it from the description of `output_diisplay_limit`.

Reply:
Done, also fixed the submissions page.